lib/Model/SelectHelper.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+declare(strict_types=1);
+/**
+ * @copyright Copyright (c) 2020 Joas Schilling <coding@schilljs.com>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace OCA\Talk\Model;
+
+use OCP\DB\QueryBuilder\IQueryBuilder;
+
+class SelectHelper {
+	public function selectRoomsTable(IQueryBuilder $query, string $alias = 'r'): void {
```

Comment:
for readability on the calling side, I think I wouldn't make the alias optional.
I came here to check as I was confused where the alias comes from

Reply:
The alias is optional?

Reply:
Ah, you would **n't**.

Well it's r for room, a for attendee and s for session. Your IDE (if you have one) will also tell you that. Not a huge fan of adding hundreds of `'(a|r|s)` in the code just for lazy non-IDE users

Reply:
it's not only for non-IDE but also when reading the code on Github for example, like when reviewing code changes or viewing a commit / diff on Github or CLI

but as you like, not a big issue to infer the alias